FSBeheer/FSBeheer/VM/QuestionnaireVM.cs

```diff
@@ -8,10 +8,11 @@
 using System.Linq;
 using System.Windows.Input;
 using System;
+using GalaSoft.MvvmLight;
 
 namespace FSBeheer.VM
 {
-    public class QuestionnaireVM
+    public class QuestionnaireVM : ViewModelBase
```

Comment:
Waarom deze overerving?

Reply:
Vanwege de Base.RaisePropertyChanged() functie die in de setters van meerdere properties worden gebruikt